delphi/app/graph.py

```diff
@@ -1,75 +1,280 @@
-"""
-Contains classes and utilities for graph operations.
-"""
 
-class GraphNode():
+def find_path(start_node, flow, people):
```

Comment:
This should probably be part of the flow network rather than a util function

Reply:
Would not make sense. Only used on the positive_flow_network variable which is a list, not a network flow

Reply:
based on the code it looks likes this is taking the results of the flow network to compute this path

Reply:
The flow network has both positive edges and negative edges (for the residual edges). I am pulling out  edges with flow > 0 and moving them in to a list to compute paths for oracle. They are no longer in the NetworkFlow when this happens. thus, find_path is not part of the network flow class

Reply:
I'm a bit confused on this, what is the network flow returning?